src/main/java/horseshoe/Settings.java

```diff
@@ -166,30 +177,21 @@ public String escape(final String raw) {
 				GregorianCalendar.class,
 				Objects.class,
 				Matcher.class,
-				Pattern.class));
-
-		if (Properties.JAVA_VERSION >= 8.0) {
-			for (final String className : Arrays.asList("java.time.Duration",
-					"java.time.Instant",
-					"java.util.Optional",
-					"java.util.OptionalDouble",
-					"java.util.OptionalInt",
-					"java.util.OptionalLong",
-					"java.util.Spliterators",
-					"java.util.stream.StreamSupport",
-					"java.util.stream.Stream",
-					"java.util.stream.DoubleStream",
-					"java.util.stream.IntStream",
-					"java.util.stream.LongStream",
-					"java.util.stream.Collector",
-					"java.util.stream.Collectors")) {
-				try {
-					defaultLoadableClasses.add(Class.forName(className));
-				} catch (final ClassNotFoundException e) {
-					Template.LOGGER.log(Level.WARNING, "Failed to load class \"{0}\", even though Java 8 or later was detected.", className);
-				}
-			}
-		}
+				Pattern.class,
+
```

Comment:
You missed Duration. Also, Thomas wanted String.

Reply:
I added Duration. String is already present.